server/services/UserService.js

```diff
@@ -39,8 +45,8 @@ class UserService {
     )
     const loggedInUser = {
       token,
-      first_name: req.user.firstName,
-      last_name: req.user.lastName,
+      first_name: req.user.first_name,
+      last_name: req.user.last_name,
```

Comment:
Make use of object destructuring

Reply:
Thank you, for the feedback 

I am working on it